loadgen/results.cc

```diff
@@ -299,15 +339,15 @@ void PerformanceSummary::LogSummary(AsyncSummary& summary) {
 
   switch (settings.scenario) {
     case TestScenario::SingleStream: {
-      summary(DoubleToString(target_latency_percentile.percentile * 100, 0) +
-                  "th percentile latency (ns) : ",
-              target_latency_percentile.sample_latency);
+      summary(DoubleToString(token_target_latency_percentile.percentile * 100, 0) +
```

Comment:
BTW, we have other benchmarks that requires sample latencies. If I am not mistaken, the changes here impact all the benchmarks, I think.

Reply:
Thanks for pointing this out. Yes, that should only be reported if we are reporting the token latencies